extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/DevServicesConfig.java

```diff
@@ -24,7 +24,7 @@
     /**
      * The container image name to use, for container based DevServices providers.
      */
-    @ConfigItem(defaultValue = "quay.io/keycloak/keycloak:14.0.0")
+    @ConfigItem(defaultValue = "quay.io/keycloak/keycloak-x:14.0.0")
```

Comment:
I think 15 is out ? Or will you upgrade that in a separate step ?

Reply:
@phillip-kruger yes, I'd like to keep the same version for both Wildfly and Quarkus distributions - so this version will get updated once I'll update the image name/version used for the integration testing in `build-parent/pom.xml`